apps/exit-node/src/index.ts

```diff
@@ -28,12 +28,15 @@ export const start = async (ops: {
         rpchRequest.provider
       );
       const rpchResponse = rpchRequest.createResponse(response);
-      await ops.hoprd.sendMessage({
-        apiEndpoint: ops.apiEndpoint,
-        apiToken: ops.apiToken,
-        message: rpchResponse.toMessage().body,
-        destination: rpchRequest.origin,
-      });
+      const segments = rpchResponse.toMessage().toSegments();
+      for (const segment of segments) {
+        await ops.hoprd.sendMessage({
```

Comment:
```suggestion
        ops.hoprd.sendMessage({
```

Reply:
let's keep the use of `hoprd.sendMessage` consistent and remove `await` similar to https://github.com/Rpc-h/RPCh/blob/a9e6b9368b7a664d8d1c42298e2dcbe7bc323b4c/packages/sdk/src/index.ts#L191-L198
we might change this later on